modules/olm-deleting-operators-from-a-cluster-using-web-console.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-adding-operators-to-cluster.adoc
+
+[id='olm-deleting-operators-from-a-cluster-using-web-console-{context}']
+= Deleting Operators from a cluster using the web console
+
+To delete an installed Operator from the selected namespace through the Web console, follow these steps:
+
+.Procedure
+
+. Navigate in the web console to the *Catalog* → *Operator Management* page.
+. Click the *Operator Subscriptions* tab to see the list of installed Operators
+for the namespace.
+. Find the Operator you want to delete (in this example, jaeger) and click
+the three-dot drop-down box in far right column.
++
+image::olm-operator-delete.png[]
+
+. Click *Remove Subscription*.
+. When prompted from the *Remove Subscription* pop-up, optionally select the
+`Also completely remove the jaeger Operator from the selected namespace`
+checkbox, if you want all components related to the installation to be removed.
+This removes the CSV, which in turn removes the pods, the deployments, the
+CRDs, and CRs associated with the Operator.
+. Select *Remove*. This Operator will stop running and no longer receive updates.
+
+[NOTE]
+====
+Although the Operator is no longer installed or receiving updates, that
+Operator will still appear on the Operator Catalogs list, ready to re-subscribe. To remove the Operator from that
+listing, you can delete the Operator's entry in the CatalogSourceConfig
+from the command line (as shown in the next section).
```

Comment:
Is this referring to the CLI section?

Reply:
Yes. We're not allowed to do xrefs in the current tooling. I'll try to find a better way to be more clear about this.